src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -306,7 +306,11 @@ def reinit_data(self):
             logger.debug("Got mgmt ip from db : {}".format(self.mgmt_ip_str))
         try:
             addr_subtype_sub_oid = 4
-            mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in self.mgmt_ip_str.split('.')])
+            mgmt_ip_sub_oid = ()
+            for mgmt_ip in self.mgmt_ip_str.split(','):
```

Comment:
>split(',') [](start = 44, length = 10)

Could you add test case for multiple ip?

Reply:
1. I cannot make sure that ipv4 address in the first place in mgmt_ip_str or not. 
	Could you help to confirm the following case and behavior?
	a. "ipv4"  
	   => Do mgmt_ip_sub_oid assignment. Can expect to succeed, if error happened, retrun.
	b. "ipv4, ipv6" 
	   ==> Do mgmt_ip_sub_oid assignment. Can expect to succeed, if error happened, retrun.
	c. "ipv6"
	   ==> Do mgmt_ip_sub_oid assignment or skip? If do mgmt_ip_sub_oid assignment, will get error and return.
	d. "ipv6, ipv4" ==> 
	    ==> Do mgmt_ip_sub_oid assignment or skip? If do mgmt_ip_sub_oid assignment, will get error and return and don't continue to parse ipv4 string

2. I don't setup snmpagent test environment yet. I can try to setup and add test case later. I will submit another pull request for test case. Thanks.
Maybe change https://github.com/Azure/sonic-snmpagent/blob/master/tests/mock_tables/appl_db.json#469

Reply:
If there is ipv4, then we will get the first ipv4 and succeed.
If there is no ipv4, we should skip all of them and keep an the MIBUpdater empty.

Please try add test case in this PR, as all other PR in this repo.

---
In reply to: [278373400](https://github.com/Azure/sonic-snmpagent/pull/106#discussion_r278373400) [](ancestors = 278373400)